README.md

```diff
@@ -1,47 +1,75 @@
-# Astronomy and Physics Society Website
+# 🌌 Astronomy and Physics Society (APS) Official Website
 
-Welcome to the APS Website repository! This project aims to create an intuitive, responsive website for the Astronomy and Physics Society, IIITDM Jabalpur. We also aim to make it an online platform for astronomy enthusiasts to connect, learn, and explore the wonders of the universe together. This website aims to provide resources, tools, and a space for members to engage in discussions, share observations, and participate in club activities.
-##
+Welcome to the official website of the **Astronomy and Physics Society (APS)** at IIITDM Jabalpur! Our platform is dedicated to uniting astronomy enthusiasts, providing information about our club, and showcasing our activities and events.
+
+## Features
+
+- **Recent Events**: Stay informed about our latest events, workshops, and stargazing sessions. We regularly update our events calendar to keep you in the loop.
+
+- **Member Information**: Learn about the passionate individuals behind APS. Our members are the driving force of our society, and we take pride in their dedication and contributions.
+
+- **Join Us**: Interested in astronomy and physics? Join our society effortlessly through our website. We welcome students and enthusiasts who share our passion for exploring the cosmos.
+
+- **Event Details**: Get comprehensive information about upcoming and past events, including schedules, topics, and participation details. We aim to provide engaging and educational experiences for all members.
+
+- **Gallery**: Explore our gallery featuring photos and highlights from our past events. From stargazing nights to guest lectures, relive the moments with us.
+
+- **Educational Outreach**: Engage in activities aimed at promoting astronomy education in schools and communities. Volunteer for events such as public stargazing sessions and school presentations, and contribute to fostering a love for astronomy among young minds.
+
+## Connect with Us
```

Comment:
Connect and contact section are overlapping.
Why did you add Hacktoberfest logo again?